sdk/keyvault/azure-keyvault-certificates/HISTORY.md

```diff
@@ -1,5 +1,39 @@
 # Release History
 
-## 4.0.0b1 (2019-06-28)
-For release notes and more information please visit
-https://aka.ms/azure-sdk-preview1-python
+## 4.0.0b3 (2019-09-10)
+Version 4.0.0b1 is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Key Vault.
+For more information about preview releases of other Azure SDK libraries, please visit https://aka.ms/azure-sdk-preview1-python.
+
+ This library is not a direct replacement for `azure-keyvault`. Applications
+using that library would require code changes to use `azure-keyvault-certificates`.
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates/samples)
+demonstrate the new API.
+
+### Breaking changes from `azure-keyvault`:
+- Packages scoped by functionality
+    - `azure-keyvault-certificates` contains a client for certificate operations
+- Client instances are scoped to vaults (an instance interacts with one vault
+only)
+- Removed `azure.core.Configuration` from the public API in preparation for a
```

Comment:
This was a b2 -> b3 change, so you can remove it here.

Reply:
got rid of config bullet, thanks!